data/human/human missions.txt

```diff
@@ -9341,3 +9342,799 @@ mission "Timothy Radrickson 7: Grave"
+			`	At the center of it all is a ravaged dead body on the floor, and above it stands a bloody grinning Ildico.`
+			choice
+				`	"Ildico, what's going on?"`
+				`	"Are you hurt? Do you need help?"`
+				`	"What have you done?"`
+				`	(Say nothing.)`
+
+			`	She ignores you, her attention focused elsewhere.`
+			`	"You thought," Ildico says as she catches her breath, "that you could sell me? That you could give me to the bureaucrat who signed Hope's death warrant? That I would be your pawn, your tool, your object in order to gain favor and promotion?"`
+			`	The raven-haired woman laughs, as she cleans off her bloody dagger, and takes a step toward a figure cowering in the corner. The man you recognize as her father.`
+			`	"Wh-what have you done? You've ruined everything. I was so close. I would have had it all." Deoric blinks, his face sets in hard, and he straightens up, he looks as if he just remembered who he is.`
+			`	"You stupid, selfish, arrogant brat. You just needed to do as you were told. To play your pathetic part. You couldn't could you? You were always worthless. If only your brother had survived." Deoric picks up a broken furniture leg and confidently wields it like a club.`
+			`	At this Ildico's wicked grin widens deeper. "My brother? Dear Hogne. Father, I killed him. He couldn't handle the ice, the isolation, the hopelessness of Hope. He lost his mind. He tried to make me his wife. And mother? She cracked long before that. An entitled debutant like her had no chance in that hellhole, and I? I needed to eat."`
+			`	The color drains out of Deoric's face. "Monster." He says. "I will put an end to you. I will start a new family, as I should have before."`
+			`	Deoric lunges at Ildico, but before he can take more then a single step, Ildico's wrist flicks and her knife slams into Deoric's throat. He falls to the ground, bleeding out, dying.`
+			`	Ildico stands over him. "You are a coward. A miserable disgusting man." She spits on him, and pulls her dagger from his heaving neck. He tries to speak as his dying body squirms, but no words materialize and he goes still.`
+			`	She turns to you. "I am now a fugitive, and you are my only friend. I need you to get us out of here."`
+			`	Ildico walks back to corner far from any of the fighting, and retrieves a bundled baby you'd neglected to notice until now.`
+			choice
+				`	"You got it. Let's go."`
+				`	"Anything for you, Ildy."`
+				`	"Sure, but you owe me. Big."`
+				`	"No way. You're a murderer. I want nothing to do with you."`
+					goto leave
+				`	"Stop right there. I am putting you under citizen's arrest until the police arrive."`
+					goto arrest
+
+			`	She nods, her face softens, and tears glitter around her eyes. "Thank you, I knew- I knew I could count on you."`
+			`	Ildico recollects herself for a moment before pointing to a briefcase lying by a table. "Grab that, and let's go."`
+			choice
+				`	"Go where?"`
```

Comment:
Add another dialog choice here, even if it just leads to the same line.

Reply:
Okay, done.

Reply:
Double doned.